src/routes/safe/components/Transactions/TxList/TxInfoSettings.tsx

```diff
@@ -85,7 +85,21 @@ export const TxInfoSettings = ({ settingsInfo }: TxInfoSettingsProps): ReactElem
         </InfoDetails>
       )
     }
+    case 'SET_GUARD': {
+      return (
+        <InfoDetails title="Set guard:">
+          <AddressInfo
+            address={settingsInfo.guard.value}
+            name={settingsInfo.guard?.name || undefined}
+            avatarUrl={settingsInfo.guard?.logoUri || undefined}
+          />
+        </InfoDetails>
+      )
+    }
+    case 'DELETE_GUARD': {
+      return <InfoDetails title="Delete guard">{null}</InfoDetails>
```

Comment:
Can we display the address of the deleted guard instead of null?

Reply:
Sadly no 'deleted guard' is returned.